news/5491.doc

```diff
@@ -0,0 +1 @@
+Use Sphinx :pep: syntax to link PEPs
```

Comment:
Could you change this NEWS fragment to a .trivial? I don't think this warrants a NEWS entry -- change with no visible effect isn't worth calling out to end-users.